css/form.css

```
@@ -1134,15 +1157,10 @@ html[dir="rtl"] .radio.radio-icon input[type="radio"] + label > span.check {
   width: 100% !important;
 }
 
-[data-widget-package="com.fliplet.form-builder"]
-  .form-group.fl-date-picker.date-picker-start
-  > .form-control,
-[data-widget-package="com.fliplet.form-builder"]
-  .form-group.fl-date-picker.date-picker-end
-  > .form-control {
-  padding-right: 0;
+[data-widget-package="com.fliplet.form-builder"] .form-group.fl-date-picker > .form-control,
+[data-widget-package="com.fliplet.form-builder"] .form-group.fl-time-picker > .form-control {
```

Perhaps this isn't necessary?

```suggestion
.form-group.fl-time-picker > .form-control {
```